integration-test/test-image/Dockerfile

```diff
@@ -1,7 +1,18 @@
-FROM hortonworks/cloudbreak
+FROM openjdk:11-jdk-slim
+MAINTAINER info@hortonworks.com
+
+WORKDIR /
+
+# Install zip
+RUN apt-get update --no-install-recommends && apt-get install -y zip procps wget && apt-get clean && rm -rf /var/lib/apt/lists/*
 
 ADD cloudbreak.jar /
+ADD start_cloudbreak_app.sh /
+ADD wait_for_cloudbreak_api.sh /
+ADD jmx_prometheus_javaagent-0.10.jar /
```

Comment:
We are using Micrometer, bundled with Spring Boot. I am not sure if this jar is still needed

Reply:
Just noticed this comment now -- in the interests of not changing too much with this change, I'm going to leave this as is and we can file a separate JIRA for removing the jar -- this JAR is still used to generate the official Cloudbreak container as well.